cmd/stack_create.go

```diff
@@ -154,8 +147,7 @@ The stack name must start with a lowercase letter, and can contain only lowercas
 
 				if stackEntryURL == "" {
 					//TODO: REMOVE OLD CREATE METHOD AFTER NEXT RELEASE AND UPDATE STACKS
-					//return errors.New("No source URL specified.  Use the add-to-repo command with the --release-url flag to your repo")
-					return oldCreateMethod(rootConfig.LoggingConfig, rootConfig, config, stack, currentTime)
+					return errors.New("No source URL specified.  Use the add-to-repo command with the --release-url flag to your repo")
```

Comment:
Could we be more specific on where the source url is supposed to be? (Also maybe remove the TODO haha)